substrate/client/transaction-pool/src/fork_aware_txpool/fork_aware_txpool.rs

```diff
@@ -469,12 +469,59 @@ where
 		self.view_store.futures_at(at)
 	}
 
+	/// Searches in the view store for a view by iterating through the fork of
+	/// the `at` block, up to the last finalized block.
+	///
+	/// Returns with a pair of a maybe view and a set of enacted blocks (possibly empty if no view
+	/// is found), usually starting at the last known block of the fork, up to the block
+	/// corresponding to the found view.
+	fn find_best_view(
+		&self,
+		at: &HashAndNumber<Block>,
+	) -> (Option<Arc<View<ChainApi>>>, Vec<Block::Hash>) {
```

Comment:
why not return option instead of tuple?
```suggestion
	) -> Option<(Arc<View<ChainApi>>, Vec<Block::Hash>)> {
```

Reply:
Right, this is definitely better. [9d47e15](https://github.com/paritytech/polkadot-sdk/pull/8533/commits/9d47e15417b4cf3fe54ec942fc883f852e494b32)